.gitignore

```diff
@@ -57,6 +57,12 @@ compile_commands.json
 /node_modules/
 /Debug
 /Release
+/modules/web
```

Comment:
There are cases when garbage appears in /modules/web and /modules/Cesium

Reply:
Does it mean that we should add this lines?

Reply:
Removed, due to possible existance of the srs in this directories